### PR TITLE
1.15 Some functions using lodash

### PR DESCRIPTION
**Pull request by AlexandraPopovaz**

### CODE REVIEW

File: src/lodash.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { capitalize, filter, find, groupBy, isNil, merge, sample, sortBy, uniq } from "lodash";
+
+function filterExample() {
+    let students = [
```
Comment (viraxslot) [inline src/lodash.ts:4]: Here and below.
const

File: src/lodash.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        { biologists: 20, girls: 5, boys: 15 },
+        { biologists: 21, girls: 20, boys: 1 },
+        { chemists: 13, girls: 4, boys: 9 },
+        { chemists: 12, girls: 2, boys: 10 },
+    ];
+    const resultFilter = filter(students, ["biologists", 21]);
```
Comment (viraxslot) [inline src/lodash.ts:10]: Here and below.

Just `return filter(students, ["biologists", 21]);`, no need to create the variable, you don't use it except return statement. It's an interesting way to filter, I've never used it. Usually it's just `filter(students, el => el.biologists === 21);` as in js standard `filter`.

File: src/lodash.ts (hunk 1 above)
Comment (AlexandraPopovaz) [reply src/lodash.ts]: Fixed

File: src/lodash.ts (hunk 2 above)
Comment (AlexandraPopovaz) [reply src/lodash.ts]: Thanks! Fixed

**Review by viraxslot — APPROVED** [review]

**Pull request merged**